packages/ui/tailwind-utils-config/components/dialog.ts

```diff
@@ -97,16 +97,21 @@ export default {
   // Body Component
   '.cn-modal-dialog-body': {
     '--cn-modal-dialog-scroll-compensation': '4px',
+    marginTop: 'var(--cn-dialog-gap)',
     paddingInline: 'var(--cn-modal-dialog-scroll-compensation)',
     paddingBottom: 'var(--cn-modal-dialog-scroll-compensation)',
     marginInline: 'calc(var(--cn-modal-dialog-scroll-compensation) * -1)',
     marginBottom: 'calc(var(--cn-modal-dialog-scroll-compensation) * -1)',
-    marginTop: 'var(--cn-dialog-gap)',
     height: '100%',
 
     '&-content': {
-      display: 'grid',
-      alignContent: 'start',
+      display: 'flex',
+      flexDirection: 'column',
+      maxWidth: 'calc(100% + var(--cn-modal-dialog-scroll-compensation)*2)',
```

Comment:
padding should be part of the width. 

`maxWidth: 'calc(100% + var(--cn-modal-dialog-scroll-compensation)*2)'` is this expected?

Reply:
Width cannot go beyond 100%. Please verify this change.